voxblox/include/voxblox/simulation/objects.h

```diff
@@ -27,19 +27,16 @@ class Object {
   virtual ~Object() {}
 
   // Map-building accessors.
-  virtual FloatingPoint getDistanceToPoint(const Point& point) const {
-    return 0.0;
-  }
+  virtual FloatingPoint getDistanceToPoint(const Point& point) const = 0;
```

Comment:
@helenol I removed the dummy implementations, this will force the descendants to implement these functions, which is what you want I guess.

Reply:
👍